lambda/create/main.go

```diff
@@ -89,12 +89,16 @@ func (l *Lambda) HandleEvent(ctx context.Context, req events.APIGatewayProxyRequ
 
 	// validation
 	if errs := Validate(input); len(errs) > 0 {
-		problem := shared.ProblemInvalidRequest
-		problem.Errors = errs
+		if input.Channel == shared.ChannelPaper {
+			l.logger.Info("encountered validation errors in lpa", slog.Any("uid", uid))
```

Comment:
what is the slightly longer term plan for these errors? are we going to store them on the document like sirius does now?